tests/help/test_help_formatter.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import annotations
+
+import textwrap
+from dataclasses import dataclass
+from typing import Literal
+
+import cappa
+import pytest
+from typing_extensions import Annotated
+
+from tests.utils import parse, strip_trailing_whitespace
+
+
+@dataclass
+class Args:
+    required: Annotated[str, cappa.Arg(help="I'm required")]
+    name: Annotated[str, cappa.Arg(help="I'm optional")] = "arg"
+    short: Annotated[str, cappa.Arg(short=True, help="I'm an option")] = "opt"
+    maybe: Annotated[str | None, cappa.Arg(short=True, help="maybe?")] = None
```

Comment:
I propably didn't read with all the attention required, nor tested the branch so apologies if that sounds silly, but why in this maybe case  `(Default: None)` is not displayed ?
what would be the difference from reading the help with `maybenodefault: Annotated[str | None, cappa.Arg(short=True, help="maybe no default?")]` ?

Reply:
I could make that bit, a bit less opinionated. I was imagining most instances where you have a `None` default end up being optional flags, where it seems less relevant (to me) to display the lack of a value as a default, you know?

It's already made clear that it's optional, and `None` is a python-specific ism, so ....idk 🤷

Reply:
your choice :) maybe others have other views on this, to be fair it's just a small "inconsistency" I noticed but overall it's not that important.